playwright/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "slicemachine-e2e",
+  "version": "0.0.1",
+  "private": true,
+  "description": "End to end tests for Slice Machine",
+  "author": "Prismic <contact@prismic.io> (https://prismic.io)",
+  "scripts": {
+    "prettier:check": "prettier --check .",
```

Comment:
We have a global .prettierrc file and then it's run by the CI in all project. So that's why SM don't have one I guess.
I wanted to follow adapters format and keep them, but we can rely on the global one if we want. I remove it actually, less config is better.

Reply:
Done no need prettier